sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -67,6 +67,18 @@ class AdaptiveQueryExecSuite
 
   setupTestData()
 
+  protected override def beforeAll(): Unit = {
+    super.beforeAll()
+    // Tests depend on intermediate results that would otherwise be cleaned up when
```

Comment:
ditto here, AQE is turned on by default now and let's make sure it's compatible with shuffle cleanup

Reply:
can we disable cleanup only for that test case?